js/ui/overlay/ui.overlay.js

```diff
@@ -256,12 +258,13 @@ const Overlay = Widget.inherit({
         this._initCloseOnOutsideClickHandler();
         this._initTabTerminatorHandler();
 
+        this._customWrapperClass = null;
         this._$wrapper = $('<div>').addClass(OVERLAY_WRAPPER_CLASS);
         this._$content = $('<div>').addClass(OVERLAY_CONTENT_CLASS);
         this._initInnerOverlayClass();
 
         const $element = this.$element();
-        this._$wrapper.addClass($element.attr('class'));
+        this.option('copyRootClassesToWrapper') && this._$wrapper.addClass($element.attr('class'));
```

Comment:
imho it's not a good idea to use `&&` instead of `if`. It worsens a code readability. 

Length of this line is 99 characters. Standard: < 80

Reply:
done